applications/nrf_desktop/doc/event_propagation.rst

```diff
@@ -823,7 +823,9 @@ nRF Desktop event propagation
 +-----------------------------------------------+-----------------------------------+               |                      |                                           |
 | :ref:`nrf_desktop_motion`                     | ``motion_event``                  |               |                      |                                           |
 +-----------------------------------------------+-----------------------------------+               |                      |                                           |
-| :ref:`nrf_desktop_usb_state`                  | ``usb_hid_event``                 |               |                      |                                           |
+| :ref:`nrf_desktop_hids`                       | ``hid_report_subscriber_event``   |               |                      |                                           |
```

Comment:
Please align with the updated arrays that were generated by me locally using a script (to keep consistency): https://github.com/MarekPieta/fw-nrfconnect-nrf/tree/kaga_new_arrays

Reply:
Have you updated the arrays ?